osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -66,6 +66,8 @@ public void UpdateBeatmap(WorkingBeatmap beatmap)
                 Depth = info?.Depth + 1 ?? 0,
             }, newInfo =>
             {
+                State = beatmap == null ? Visibility.Hidden : Visibility.Visible;

```

Comment:
Is this going to conflict with the this.FadeIn() call right below?

Reply:
Something like _entering_ songselect with no beatmaps?

Reply:
This one is scheduled so takes precedence. Probably don't need the FadeIn below though